src/cli/bin.js

```diff
@@ -2,6 +2,7 @@
 
 'use strict'
 
+require('pipe-args').load()
```

Comment:
@diasdavid being /dev/stdin a symbolic link to /proc/self/fd/0 (and /proc/self a symlink to the running process), we want to grab whatever it is there and pass it as argument, right? The problem I see is `echo "file.sx file.json" | ipfs add` will not work since the input 'filesx file.json' will be passed as one string. Or I am missing something else?